### ISSUE TITLE
wav upload doesn't work

### ISSUE THREAD

**Issue by KeenVox** [ChrisMicro/AttinySound#2]
Hi,
I can't get the code upload via wav to work. I triple checked my wiring, the fuse setting and the avr commands.

I can send sound to the Attiny and the led to show if it receives data blinks but after uploading nothing happens. 

If I programm the Attiny over ISP every programm works flawlessly.

I know this is an old project, but maybe someone can give me any tipps where to troubleshoot?

Thx

**Comment by ChrisMicro** [comment]
Did you try to adjust the loudness of the sound? A good level is normally 3/4 max.
Probably you should also try it with another computer. With newer Apple computers problems occur (might be due to some special filters which are not standard in other computer outputs).

**Comment by KeenVox** [comment]
Yeah, tried from 30-100%,  nothing seems to work. Tried from my phone, but didn't work either. 
I can see it receiving the data, but nothing sticks. I would try programming the attiny the normal way, but I can't find the code for everything. :-(

**Comment by ChrisMicro** [comment]
I had some problems when using the windows media player to play the sound files because it introduces a slow "fade in" at the beginning of the sound which confuses the bootloader.

**Comment by KeenVox** [comment]
Hm, I tried on the sample page, and with winamp. Will try a different player...
